include/misc/polyfill-each.php

```diff
@@ -0,0 +1,54 @@
+<?php
+// MIT License
```

Comment:
is this license compatible with phorum/core to be bundled?

Reply:
Phorum license is based on The Apache Software License Version 1.1. Both licenses are permissive, non-copyleft. So I think it's okay. [Some explanation](https://law.stackexchange.com/questions/6081/can-i-bundle-mit-licensed-components-in-a-apache-2-0-licensed-project).